flink-connectors/flink-connector-hbase-base/src/main/java/org/apache/flink/connector/hbase/sink/HBaseSinkFunction.java

```diff
@@ -191,8 +195,10 @@ private void checkErrorAndRethrow() {
     public void invoke(T value, Context context) throws Exception {
         checkErrorAndRethrow();
 
-        mutator.mutate(mutationConverter.convertToMutation(value));
-
+        Mutation mutation = mutationConverter.convertToMutation(value);
+        synchronized (mutationMap) {
+            mutationMap.put(mutation.getRow(), mutation);
```

Comment:
In this way, all mutations with the same rowkey value will only remains the last one.
But `mutationConverter` behavior is not controlled, which means such mutations list will casuse data quaility problem.
`
-D (rk1, f1:v1)     
+I (rk1, f2:v2)
`

Reply:
For now, all mutationConverter implementations is ok in this case.